modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -523,6 +523,8 @@ class ApiClient(object):
             if auth_setting:
                 if not auth_setting['value']:
                     continue
+                elif auth_setting['in'] == 'cookie':
+                    headers['Cookie'] = auth_setting['value']
```

Comment:
Should it be something like:
`headers['Cookie'] = f"{auth_setting['cookie_key']}={auth_setting['value']}"`?

Cookies have keys in front of them usually, no?